src/components/pages/ComparisonPage/ComparisonRow.tsx

```diff
@@ -4,11 +4,30 @@ import styles from "./ComparisonRow.module.scss";
 
 type Props = {
   children: JSX.Element[];
+  emptyCellsText?: JSX.Element;
 };
 
 function ComparisonRow({ children }: Props) {
   const rowLength = children.length;
-  return <div className={`${styles.row_style}`}>{children}</div>;
+  const emptyCellsText = "Legg til yrke for å sammenligne tallene";
```

Comment:
Translate?

Reply:
Jada, men da bør jeg vel også sørge for "yrke" vs "utdanning"

Reply:
Skulle gjerne formulert noe som ikke krevde innholdstype

Reply:
Nå viser den også yrke selv om jeg har valgt utdannelse